model/src/main/scala/lucuma/schemas/model/ScienceMode.scala

```diff
@@ -0,0 +1,269 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.schemas.model
+
+import cats.Eq
+import cats.data.NonEmptyList
+import cats.derived.*
+import cats.syntax.all.*
+import eu.timepit.refined.cats.*
+import io.circe.Decoder
+import io.circe.generic.semiauto.*
+import io.circe.refined.*
+import lucuma.core.enums.*
+import lucuma.core.math.Offset
+import lucuma.core.math.Wavelength
+import lucuma.core.math.WavelengthDither
+import lucuma.schemas.decoders.given
+import monocle.Focus
+import monocle.Lens
+import monocle.Prism
+import monocle.macros.GenPrism
+
+sealed abstract class ScienceMode(val instrument: Instrument) extends Product with Serializable
```

Comment:
Would this be a good time to rename this to `ObservingMode` to match the API?

Reply:
Definitely, let me do that.